Misc/NEWS.d/next/Library/2019-05-16-23-40-36.bpo-24564.lIwV_7.rst

```diff
@@ -0,0 +1,3 @@
+shutil.copystat(): ignore EINVAL on os.setxattr() which may occur when copying files on NFS filesystems. 

```

Comment:
Do you mind adding markup and reword the NEWs entry? I think something like that could be good (think to use `blurb` to format correctly the lines):
```
:func:`shutil.copystat` now ignores :const:`errno.EINVAL` on :func:`os.setxattr` which may occur when copying files on filesystems without extended attributes support. Original patch by Giampaolo Rodola, updated by Ying Wang.
```

Reply:
Sounds great, should be pushed 👍